src/SFA.DAS.Forecasting.Application/Projections/Services/AccountProjectionDataSession.cs

```diff
@@ -131,7 +140,11 @@ public async Task SaveChanges()
             var stopwatch = new Stopwatch();
             stopwatch.Start();
             var startTime = DateTime.UtcNow;
-            await _dataContext.SaveChangesAsync();
+            using (var scope = new TransactionScope(TransactionScopeOption.Required, new TransactionOptions { IsolationLevel = IsolationLevel.Snapshot }, TransactionScopeAsyncFlowOption.Enabled))
```

Comment:
@abroodie 
Do we need **SaveChanges** here as we are using  **ExecuteSqlCommandAsync**  for store and delete❓ 
/c